Lib/glyphsLib/classes.py

```diff
@@ -3448,6 +3459,8 @@ def shouldWriteValueForKey(self, key):
     def _setupLayer(self, layer, key):
         assert isinstance(key, str)
         layer.parent = self
+        if layer.hasBackground:
+            layer._background.parent = self
```

Comment:
Is this still needed given the other changes?

Reply:
This is the change that actually makes tests pass.